query_execution/WorkerMessage.hpp

```diff
@@ -37,7 +37,7 @@ class WorkerMessage {
   };
 
   /**
-   * @brief A static named constructor for generating rebuild WorkOrder messages.
+   * @brief A static factory method for generating rebuild WorkOrder messages.
```

Comment:
Need @return doxygen for all the three methods. Earlier it was a constructor, so we didn't need it.